packages/manager/modules/manager-pci-common/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "@ovh-ux/manager-pci-common",
+  "version": "0.0.0",
+  "private": true,
+  "sideEffects": false,
+  "type": "module",
+  "main": "./src/index.ts",
+  "scripts": {
+    "coverage": "vitest run --coverage",
+    "dev": "tsc",
+    "lint": "eslint ./src",
+    "test": "vitest run"
+  },
+  "dependencies": {
+    "@ovh-ux/manager-config": "^7.3.2",
+    "@ovh-ux/manager-core-api": "^0.7.1",
+    "@ovh-ux/manager-react-shell-client": "^0.6.1",
+    "@ovh-ux/manager-tailwind-config": "^0.2.0",
+    "@ovh-ux/shell": "^3.6.1",
+    "@ovhcloud/manager-components": "^1.19.0",
+    "@ovhcloud/ods-common-core": "^17.2.2",
+    "@ovhcloud/ods-common-stencil": "^17.2.2",
+    "@ovhcloud/ods-common-theming": "^17.2.2",
+    "@ovhcloud/ods-components": "^17.2.2",
+    "@ovhcloud/ods-theme-blue-jeans": "^17.2.2",
+    "@tanstack/react-query": "^5.37.1",
+    "@tanstack/react-query-devtools": "^5.37.1",
+    "@tanstack/react-table": "^8.17.3",
+    "clsx": "2.1.1",
+    "date-fns": "^3.3.1",
+    "element-internals-polyfill": "^1.3.10",
+    "file-saver": "^2.0.5",
+    "i18next": "^23.8.2",
+    "i18next-http-backend": "^2.4.3",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-hook-form": "^7.50.1",
+    "react-i18next": "^14.0.5",
+    "react-router-dom": "^6.22.0",
+    "react-use": "^17.5.0"
```

Comment:
Can we mention `react`, `react-dom`, `tanstack`, `ods` and other as peerDependendency to avoid having multiple versions of the packages?